recipes/dask-jobqueue/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "dask-jobqueue" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "4ad6dc9939ef0d2b8b0632ef70b9084277a4de2943b4ea29d73056a4e8f92b07" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - dask_jobqueue
+
+about:
+  home: https://github.com/dask/dask-jobqueue
+  license: BSD 3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
+  description: |
+    Easy deployment of Dask Distributed on job queuing systems such as PBS,
+    Slurm, or SGE.
+  doc_url: http://simplejson.readthedocs.io/
+  dev_url: https://github.com/simplejson/simplejson
```

Comment:
This needs to be replaced with the actual url

Reply:
thanks for the catch!